gecode/int/branch.hh

```diff
@@ -672,6 +672,123 @@ namespace Gecode { namespace Int { namespace Branch {
 
 #include <gecode/int/branch/view-values.hpp>
 
+#ifdef GECODE_HAS_CBS
```

Comment:
Is gecode/int/branch.hh a good place to put the definition of my brancher?

Reply:
Yes, absolutely!